kolibri/content/utils/channel_import.py

```diff
@@ -277,6 +281,22 @@ def generate_local_file_from_file(self, SourceRecord):
     def set_version_to_no_version(self, source_object):
         return NO_VERSION
 
+    def get_license(self, SourceRecord):
```

Comment:
This code is necessary to be backwards compatible with old content databases, correct?

Reply:
Yeah, this imports license data from old database versions into the new denormalized format.